### PR TITLE
Enabling metrics exchange with prometheus

### PR DESCRIPTION
**Pull request by SamuelStuchly**
work in progress, do not merge,  added manifests service and service-monitor to enable prometheus to get metrics from insights operator

### CODE REVIEW

**Comment by SamuelStuchly** [comment]
It is finished, ready to be merged. Tested on openshift cluster generated by cluster-bot app. Metrics from insights-operator are now visible in Prometheus.

File: manifests/03-clusterrole.yaml [hunk 1]
```diff
@@ -195,3 +195,4 @@ subjects:
 - kind: ServiceAccount
   name: operator
   namespace: openshift-insights
+
```
Comment (iNecas) [inline manifests/03-clusterrole.yaml]: Nitpick: this whitespace change looks like unnecessary

File: manifests/07-servicemonitor.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    interval: 30s
+    metricRelabelings:
+    - action: drop
+      regex: etcd_(debugging|disk|request|server).*
+      sourceLabels:
+      - __name__
+    port: https
+    scheme: https
+    tlsConfig:
+      # TODO where do you mount the service-ca.crt?
```
Comment (iNecas) [inline manifests/07-servicemonitor.yaml]: How about this TODO? I see other operators having handled this https://github.com/openshift/cluster-authentication-operator/blob/3ccaf76d5a1f139800fd2360fd2d52af2edd4c35/manifests/0000_90_cluster-authentication-operator_02_servicemonitor.yaml#L20

File: manifests/03-clusterrole.yaml (hunk 1 above)
Comment (tisnik) [reply manifests/03-clusterrole.yaml]: probably the wrong text editor configuration

**Comment by iNecas** [comment]
Tested and worked well.

/lgtm

**Comment by iNecas** [comment]
Thanks @SamuelStuchly

**Pull request merged**